web/static/ivre/graph.js

```diff
@@ -268,6 +272,11 @@ var GraphTopValues = (function(_super) {
 		    return 'setparam(FILTER, "script", "' + x + '");';
 		};
 	    }
+            else if(field.substr(0, 7) === 'script:') {
+                preparefilter = function(x) {
+                    return 'setparam(FILTER, "'+ field +'", "/' + escapeRegex(x) + '/");setparam(FILTER, "display:'+ field +'");';
```

Comment:
I don't think we should have the `display:` directive.
```suggestion
                    return 'setparam(FILTER, "'+ field +'", "/' + escapeRegExp(x) + '/");
```

Reply:
I understand but that's not what exists in IVRE so far. For now, clicking in top values creates a filter for that value, it does not limit the display. For now, I'd rather keep it that way.

Reply:
> I understand but that's not what exists in IVRE so far. For now, clicking in top values creates a filter for that value, it does not limit the display. For now, I'd rather keep it that way.

No problem, you're the boss, boss ;)